### PR TITLE
Better linting

### PR DESCRIPTION
**Pull request by dsiddharth**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,7 +47,9 @@
     "@typescript-eslint/parser": "5.26.0",
     "eslint": "8.16.0",
     "eslint-plugin-import": "2.26.0",
+    "eslint-plugin-prettier": "^4.0.0",
     "npm-package-json-lint": "6.3.0",
+    "prettier": "^2.6.2",
```
Comment (hpx7) [inline package.json]: remove carets

Comment (dsiddharth) [reply package.json]: how did this pass lint checks?

Comment (dsiddharth) [reply package.json]: the change in the PR broke it, but it should work now

Comment (dsiddharth) [reply package.json]: yep, here's the error now: https://github.com/hathora/hathora/runs/6757009799?check_suite_focus=true

**Pull request merged**